docker-compose-delhi-0.7.0.yml

```diff
@@ -0,0 +1,502 @@
+# /*******************************************************************************
```

Comment:
you should consider renaming this to `docker-compose-edinburgh-1.0.0.yml` and adjusting the version numbers of the containers here

Reply:
OK applied